stock_dashboard_api/__init__.py

```diff
@@ -2,14 +2,12 @@
 
 from flask import Flask
 
-from stock_dashboard_api.config import DevelopmentConfig
 from stock_dashboard_api.views import stock_view, dashboard_views, stocks_data_view, stock_dashboard_view
 
 TEMPLATE_FOLDER = os.path.join(os.environ.get('PROD_ROOT'), 'templates')
 app = Flask(__name__, template_folder=TEMPLATE_FOLDER)
 
-
-app.config.from_object(DevelopmentConfig)
+app.config.from_object(os.environ.get('FLASK_DEVELOPMENT_CONFIG'))
```

Comment:
What if I want to use non DEVELOPMENT config, like stage or prod? Where can I configure which config should I use?

Reply:
FLASK_DEVELOPMENT_CONFIG changed to FLASK_CONFIG